roles/openshift_storage_glusterfs/defaults/main.yml

```diff
@@ -6,6 +6,7 @@ openshift_storage_glusterfs_nodeselector: "glusterfs={{ openshift_storage_gluste
 openshift_storage_glusterfs_use_default_selector: False
 openshift_storage_glusterfs_storageclass: True
 openshift_storage_glusterfs_storageclass_default: False
+openshift_storage_glusterfs_storageclassblock_default: False
```

Comment:
Rename this to `openshift_storage_glusterfs_block_storageclass_default`, move it down to the bottom of the other `_block_` variables, and create a `_registry_` version. Also implement a `openshift_storage_glusterfs_block_storageclass` variable.